README.md

```diff
@@ -1 +1,28 @@
-# dspace-submission-service
\ No newline at end of file
+# DSpace Submission Service
+
+A service for creating DSpace records and attaching metadata
```

Comment:
Should we clarify that (at least for now) it is specifically creating Item records?

Reply:
I... don't understand the difference in this context.

Reply:
Item records as opposed to Collection or Community records.